packages/main/cypress/specs/MessageStrip.cy.ts

```diff
@@ -1,8 +1,92 @@
 import { html } from "lit";
 import "../../src/MessageStrip.js";
-import { MESSAGE_STRIP_CLOSE_BUTTON_INFORMATION, MESSAGE_STRIP_CLOSE_BUTTON_CUSTOM } from "../../src/generated/i18n/i18n-defaults.js";
+import {
+	MESSAGE_STRIP_CLOSE_BUTTON_INFORMATION,
+	MESSAGE_STRIP_CLOSE_BUTTON_CUSTOM,
+	MESSAGE_STRIP_INFORMATION,
+	MESSAGE_STRIP_CLOSABLE,
+	MESSAGE_STRIP_CUSTOM,
+} from "../../src/generated/i18n/i18n-defaults.js";
+
+describe("API", () => {
+	it("tests close event", () => {
+		cy.mount(html`<ui5-message-strip>MessageStrip</ui5-message-strip>`);
+
+		cy.get("[ui5-message-strip]").then($strip => {
+			$strip.get(0).addEventListener("close", cy.stub().as("close"));
+		});
+
+		cy.get("[ui5-message-strip]")
+			.shadow()
+			.find(".ui5-message-strip-close-button")
+			.as("closeButton");
+
+		cy.get("@closeButton")
+			.click();
+
+		cy.get("@closeButton")
+			.realPress("Space");
+
+		cy.get("@closeButton")
+			.realPress("Enter");
+
+		cy.get("@close").should("have.callCount", 3);
+	});
+
+	it("test design property", () => {
+		// Arrange
+		cy.mount(html`
+			<div>
+					<ui5-message-strip id="colorSet1ColorScheme1" design="ColorSet1">MessageStrip 1</ui5-message-strip>
+					<ui5-message-strip id="colorSet1ColorScheme2" design="ColorSet2">MessageStrip 2</ui5-message-strip>
+			</div>
+		`);
+
+		// Act & Assert
+		cy.get("#colorSet1ColorScheme1").should("have.attr", "design", "ColorSet1");
```

Comment:
this test is testing framework logic. probably it is redundant.

Reply:
yeah, this functionality is mostly colors, visual tests would be a nice replacement :)